packages/ui/src/domains/LandingPageBuilder/LandingPageBuilder.tsx

```
@@ -0,0 +1,2509 @@
+	defaultSize,
+	controls,
+	onUpdate,
+}: {
+	src: string;
+	width: number;
+	height: number;
+	autoplay: boolean;
+	muted: boolean;
+	defaultSize: boolean;
+	controls: boolean;
+	onUpdate: (newProps: any) => void;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)